transform/relations/relations_utils.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+from schemas.properties.relations import DIRECTORY, FILE, MAIN_DATA_DIRECTORY
+
+
+def create_file_path_column(df):
```

Comment:
Functions in that file may be moved into the general utils directory if they would be more described

Reply:
I added type hints, docstrings, and moved functions to utils/utils.